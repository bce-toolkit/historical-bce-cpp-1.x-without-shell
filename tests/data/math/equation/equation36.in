8 6
2 0 0 0 0 -1 0 0
2 0 2 0 0 -2 -2 0
0 1 0 -2 0 0 0 0
0 1 0 0 -1 0 0 0
0 4 4 -4 -4 -2 -1 0
0 0 1 -1 -1 0 0 0
# C2H2+KMnO4+H2SO4=K2SO4+MnSO4+HCOOH+H2O
