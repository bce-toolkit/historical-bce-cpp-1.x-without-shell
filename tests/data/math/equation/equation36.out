5/4Xa
3/2Xa
9/4Xa
3/4Xa
3/2Xa
5/2Xa
Xa
